.idea/gradle.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
remove this file from VCS pls

Reply:
1b9f46e8bb52102855116719e4019693c62b3688